webapi/webapi-contactsmanager-w3c-tests/contactsmanager/ContactsManager_save_Contact_fullset.html

```diff
@@ -68,7 +68,26 @@
     preferred: true,
     value: 'msn:kangx.xu@intel.com'
   });
-  var data = new Date(1987,10,27);
+  Date.prototype.format = function(format) {
```

Comment:
To make things easier, I think we can use 'Date.prototype.getTime', rather than your format or 'toString'.

If two date objects get the same time by getTime, they are equal. Make sense?